ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/ChainHeadTrackerTest.java

```diff
@@ -56,19 +55,15 @@ public class ChainHeadTrackerTest {
 
   private final TrailingPeerLimiter trailingPeerLimiter = mock(TrailingPeerLimiter.class);
 
-  @Parameterized.Parameters
-  public static Collection<Object[]> data() {
-    return Arrays.asList(new Object[][] {{DataStorageFormat.BONSAI}, {DataStorageFormat.FOREST}});
+  static class ChainHeadTrackerTestArguments implements ArgumentsProvider {
```

Comment:
Can we just use `@ValueSource` instead?

Reply:
`@ValueSource` only allows literals https://junit.org/junit5/docs/5.1.1/api/org/junit/jupiter/params/provider/ValueSource.html

Reply:
ah that's annoying - so be it :)